cocos/2d/CCSprite.h

```diff
@@ -451,6 +451,13 @@ class CC_DLL Sprite : public Node, public TextureProtocol
      * @param PolygonInfo the polygon information object
      */
     void setPolygonInfo(const PolygonInfo& info);
+
+    /** whether or not contentSize streches the sprite's texture */
+    void setStrechEnabled(bool enabled);
+
+    /** returns whether or not contentSize streches the sprite's texture */
+    bool getStrechEnabled() const;
```

Comment:
getStrechEnabled -> isStrenchEnabled for the getter funtions which return bool value.